<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CustomConfigWidget</class>
 <widget class="QWidget" name="CustomConfigWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>880</width>
    <height>608</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_10">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_14">
     <property name="rightMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>Curve 1</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="topMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Source:</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="toolTip">
             <string>The source of Curve 1 will always be Throttle</string>
            </property>
            <item>
             <property name="text">
              <string>Throttle</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="MixerCurve" name="customThrottle1Curve" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
            <horstretch>1</horstretch>
            <verstretch>1</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>1000</width>
            <height>1000</height>
           </size>
          </property>
          <property name="sizeIncrement">
           <size>
            <width>10</width>
            <height>10</height>
           </size>
          </property>
          <property name="baseSize">
           <size>
            <width>300</width>
            <height>350</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_5">
       <property name="title">
        <string>Curve 2</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_17">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="topMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Source:</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QComboBox" name="curve2SourceCombo">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="MixerCurve" name="customThrottle2Curve" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
            <horstretch>1</horstretch>
            <verstretch>1</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>1000</width>
            <height>1000</height>
           </size>
          </property>
          <property name="sizeIncrement">
           <size>
            <width>10</width>
            <height>10</height>
           </size>
          </property>
          <property name="baseSize">
           <size>
            <width>300</width>
            <height>350</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QTableWidget" name="customMixerTable">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::NoSelection</enum>
       </property>
       <property name="columnCount">
        <number>12</number>
       </property>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>50</number>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>false</bool>
       </attribute>
       <row>
        <property name="text">
         <string>Type</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>Curve 1</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>Curve 2</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>Roll</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>Pitch</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>Yaw</string>
        </property>
       </row>
       <column>
        <property name="text">
         <string>Ch 1</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Ch 2</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Ch 3</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Ch 4</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Ch 5</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Ch 6</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Ch 7</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Ch 8</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Ch 9</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Ch 10</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Ch 11</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Ch 12</string>
        </property>
       </column>
       <item row="0" column="0">
        <property name="text">
         <string>-</string>
        </property>
        <property name="textAlignment">
         <set>AlignHCenter|AlignVCenter|AlignCenter</set>
        </property>
       </item>
       <item row="0" column="1">
        <property name="text">
         <string>-</string>
        </property>
        <property name="textAlignment">
         <set>AlignHCenter|AlignVCenter|AlignCenter</set>
        </property>
       </item>
       <item row="0" column="2">
        <property name="text">
         <string>-</string>
        </property>
        <property name="textAlignment">
         <set>AlignHCenter|AlignVCenter|AlignCenter</set>
        </property>
       </item>
       <item row="0" column="3">
        <property name="text">
         <string>-</string>
        </property>
        <property name="textAlignment">
         <set>AlignHCenter|AlignVCenter|AlignCenter</set>
        </property>
       </item>
       <item row="0" column="4">
        <property name="text">
         <string>-</string>
        </property>
        <property name="textAlignment">
         <set>AlignHCenter|AlignVCenter|AlignCenter</set>
        </property>
       </item>
       <item row="0" column="5">
        <property name="text">
         <string>-</string>
        </property>
        <property name="textAlignment">
         <set>AlignHCenter|AlignVCenter|AlignCenter</set>
        </property>
       </item>
       <item row="0" column="6">
        <property name="text">
         <string>-</string>
        </property>
        <property name="textAlignment">
         <set>AlignHCenter|AlignVCenter|AlignCenter</set>
        </property>
       </item>
       <item row="0" column="7">
        <property name="text">
         <string>-</string>
        </property>
        <property name="textAlignment">
         <set>AlignHCenter|AlignVCenter|AlignCenter</set>
        </property>
       </item>
       <item row="0" column="8">
        <property name="text">
         <string>-</string>
        </property>
        <property name="textAlignment">
         <set>AlignHCenter|AlignVCenter|AlignCenter</set>
        </property>
       </item>
       <item row="0" column="9">
        <property name="text">
         <string>-</string>
        </property>
        <property name="textAlignment">
         <set>AlignHCenter|AlignVCenter|AlignCenter</set>
        </property>
       </item>
       <item row="0" column="10">
        <property name="text">
         <string>-</string>
        </property>
        <property name="textAlignment">
         <set>AlignHCenter|AlignVCenter|AlignCenter</set>
        </property>
       </item>
       <item row="0" column="11">
        <property name="text">
         <string>-</string>
        </property>
        <property name="textAlignment">
         <set>AlignHCenter|AlignVCenter|AlignCenter</set>
        </property>
       </item>
       <item row="1" column="0">
        <property name="text">
         <string>-</string>
        </property>
        <property name="textAlignment">
         <set>AlignHCenter|AlignVCenter|AlignCenter</set>
        </property>
       </item>
       <item row="1" column="1">
        <property name="text">
         <string>-</string>
        </property>
        <property name="textAlignment">
         <set>AlignHCenter|AlignVCenter|AlignCenter</set>
        </property>
       </item>
       <item row="1" column="2">
        <property name="text">
         <string>-</string>
        </property>
        <property name="textAlignment">
         <set>AlignHCenter|AlignVCenter|AlignCenter</set>
        </property>
       </item>
       <item row="1" column="3">
        <property name="text">
         <string>-</string>
        </property>
        <property name="textAlignment">
         <set>AlignHCenter|AlignVCenter|AlignCenter</set>
        </property>
       </item>
       <item row="1" column="4">
        <property name="text">
         <string>-</string>
        </property>
        <property name="textAlignment">
         <set>AlignHCenter|AlignVCenter|AlignCenter</set>
        </property>
       </item>
       <item row="1" column="5">
        <property name="text">
         <string>-</string>
        </property>
        <property name="textAlignment">
         <set>AlignHCenter|AlignVCenter|AlignCenter</set>
        </property>
       </item>
       <item row="1" column="6">
        <property name="text">
         <string>-</string>
        </property>
        <property name="textAlignment">
         <set>AlignHCenter|AlignVCenter|AlignCenter</set>
        </property>
       </item>
       <item row="1" column="7">
        <property name="text">
         <string>-</string>
        </property>
        <property name="textAlignment">
         <set>AlignHCenter|AlignVCenter|AlignCenter</set>
        </property>
       </item>
       <item row="1" column="8">
        <property name="text">
         <string>-</string>
        </property>
        <property name="textAlignment">
         <set>AlignHCenter|AlignVCenter|AlignCenter</set>
        </property>
       </item>
       <item row="1" column="9">
        <property name="text">
         <string>-</string>
        </property>
        <property name="textAlignment">
         <set>AlignHCenter|AlignVCenter|AlignCenter</set>
        </property>
       </item>
       <item row="1" column="10">
        <property name="text">
         <string>-</string>
        </property>
        <property name="textAlignment">
         <set>AlignHCenter|AlignVCenter|AlignCenter</set>
        </property>
       </item>
       <item row="1" column="11">
        <property name="text">
         <string>-</string>
        </property>
        <property name="textAlignment">
         <set>AlignHCenter|AlignVCenter|AlignCenter</set>
        </property>
       </item>
       <item row="2" column="0">
        <property name="text">
         <string>-</string>
        </property>
        <property name="textAlignment">
         <set>AlignHCenter|AlignVCenter|AlignCenter</set>
        </property>
       </item>
       <item row="2" column="1">
        <property name="text">
         <string>-</string>
        </property>
        <property name="textAlignment">
         <set>AlignHCenter|AlignVCenter|AlignCenter</set>
        </property>
       </item>
       <item row="2" column="2">
        <property name="text">
         <string>-</string>
        </property>
        <property name="textAlignment">
         <set>AlignHCenter|AlignVCenter|AlignCenter</set>
        </property>
       </item>
       <item row="2" column="3">
        <property name="text">
         <string>-</string>
        </property>
        <property name="textAlignment">
         <set>AlignHCenter|AlignVCenter|AlignCenter</set>
        </property>
       </item>
       <item row="2" column="4">
        <property name="text">
         <string>-</string>
        </property>
        <property name="textAlignment">
         <set>AlignHCenter|AlignVCenter|AlignCenter</set>
        </property>
       </item>
       <item row="2" column="5">
        <property name="text">
         <string>-</string>
        </property>
        <property name="textAlignment">
         <set>AlignHCenter|AlignVCenter|AlignCenter</set>
        </property>
       </item>
       <item row="2" column="6">
        <property name="text">
         <string>-</string>
        </property>
        <property name="textAlignment">
         <set>AlignHCenter|AlignVCenter|AlignCenter</set>
        </property>
       </item>
       <item row="2" column="7">
        <property name="text">
         <string>-</string>
        </property>
        <property name="textAlignment">
         <set>AlignHCenter|AlignVCenter|AlignCenter</set>
        </property>
       </item>
       <item row="2" column="8">
        <property name="text">
         <string>-</string>
        </property>
        <property name="textAlignment">
         <set>AlignHCenter|AlignVCenter|AlignCenter</set>
        </property>
       </item>
       <item row="2" column="9">
        <property name="text">
         <string>-</string>
        </property>
        <property name="textAlignment">
         <set>AlignHCenter|AlignVCenter|AlignCenter</set>
        </property>
       </item>
       <item row="2" column="10">
        <property name="text">
         <string>-</string>
        </property>
        <property name="textAlignment">
         <set>AlignHCenter|AlignVCenter|AlignCenter</set>
        </property>
       </item>
       <item row="2" column="11">
        <property name="text">
         <string>-</string>
        </property>
        <property name="textAlignment">
         <set>AlignHCenter|AlignVCenter|AlignCenter</set>
        </property>
       </item>
       <item row="3" column="0">
        <property name="text">
         <string>-</string>
        </property>
        <property name="textAlignment">
         <set>AlignHCenter|AlignVCenter|AlignCenter</set>
        </property>
       </item>
       <item row="3" column="1">
        <property name="text">
         <string>-</string>
        </property>
        <property name="textAlignment">
         <set>AlignHCenter|AlignVCenter|AlignCenter</set>
        </property>
       </item>
       <item row="3" column="2">
        <property name="text">
         <string>-</string>
        </property>
        <property name="textAlignment">
         <set>AlignHCenter|AlignVCenter|AlignCenter</set>
        </property>
       </item>
       <item row="3" column="3">
        <property name="text">
         <string>-</string>
        </property>
        <property name="textAlignment">
         <set>AlignHCenter|AlignVCenter|AlignCenter</set>
        </property>
       </item>
       <item row="3" column="4">
        <property name="text">
         <string>-</string>
        </property>
        <property name="textAlignment">
         <set>AlignHCenter|AlignVCenter|AlignCenter</set>
        </property>
       </item>
       <item row="3" column="5">
        <property name="text">
         <string>-</string>
        </property>
        <property name="textAlignment">
         <set>AlignHCenter|AlignVCenter|AlignCenter</set>
        </property>
       </item>
       <item row="3" column="6">
        <property name="text">
         <string>-</string>
        </property>
        <property name="textAlignment">
         <set>AlignHCenter|AlignVCenter|AlignCenter</set>
        </property>
       </item>
       <item row="3" column="7">
        <property name="text">
         <string>-</string>
        </property>
        <property name="textAlignment">
         <set>AlignHCenter|AlignVCenter|AlignCenter</set>
        </property>
       </item>
       <item row="3" column="8">
        <property name="text">
         <string>-</string>
        </property>
        <property name="textAlignment">
         <set>AlignHCenter|AlignVCenter|AlignCenter</set>
        </property>
       </item>
       <item row="3" column="9">
        <property name="text">
         <string>-</string>
        </property>
        <property name="textAlignment">
         <set>AlignHCenter|AlignVCenter|AlignCenter</set>
        </property>
       </item>
       <item row="3" column="10">
        <property name="text">
         <string>-</string>
        </property>
        <property name="textAlignment">
         <set>AlignHCenter|AlignVCenter|AlignCenter</set>
        </property>
       </item>
       <item row="3" column="11">
        <property name="text">
         <string>-</string>
        </property>
        <property name="textAlignment">
         <set>AlignHCenter|AlignVCenter|AlignCenter</set>
        </property>
       </item>
       <item row="4" column="0">
        <property name="text">
         <string>-</string>
        </property>
        <property name="textAlignment">
         <set>AlignHCenter|AlignVCenter|AlignCenter</set>
        </property>
       </item>
       <item row="4" column="1">
        <property name="text">
         <string>-</string>
        </property>
        <property name="textAlignment">
         <set>AlignHCenter|AlignVCenter|AlignCenter</set>
        </property>
       </item>
       <item row="4" column="2">
        <property name="text">
         <string>-</string>
        </property>
        <property name="textAlignment">
         <set>AlignHCenter|AlignVCenter|AlignCenter</set>
        </property>
       </item>
       <item row="4" column="3">
        <property name="text">
         <string>-</string>
        </property>
        <property name="textAlignment">
         <set>AlignHCenter|AlignVCenter|AlignCenter</set>
        </property>
       </item>
       <item row="4" column="4">
        <property name="text">
         <string>-</string>
        </property>
        <property name="textAlignment">
         <set>AlignHCenter|AlignVCenter|AlignCenter</set>
        </property>
       </item>
       <item row="4" column="5">
        <property name="text">
         <string>-</string>
        </property>
        <property name="textAlignment">
         <set>AlignHCenter|AlignVCenter|AlignCenter</set>
        </property>
       </item>
       <item row="4" column="6">
        <property name="text">
         <string>-</string>
        </property>
        <property name="textAlignment">
         <set>AlignHCenter|AlignVCenter|AlignCenter</set>
        </property>
       </item>
       <item row="4" column="7">
        <property name="text">
         <string>-</string>
        </property>
        <property name="textAlignment">
         <set>AlignHCenter|AlignVCenter|AlignCenter</set>
        </property>
       </item>
       <item row="4" column="8">
        <property name="text">
         <string>-</string>
        </property>
        <property name="textAlignment">
         <set>AlignHCenter|AlignVCenter|AlignCenter</set>
        </property>
       </item>
       <item row="4" column="9">
        <property name="text">
         <string>-</string>
        </property>
        <property name="textAlignment">
         <set>AlignHCenter|AlignVCenter|AlignCenter</set>
        </property>
       </item>
       <item row="4" column="10">
        <property name="text">
         <string>-</string>
        </property>
        <property name="textAlignment">
         <set>AlignHCenter|AlignVCenter|AlignCenter</set>
        </property>
       </item>
       <item row="4" column="11">
        <property name="text">
         <string>-</string>
        </property>
        <property name="textAlignment">
         <set>AlignHCenter|AlignVCenter|AlignCenter</set>
        </property>
       </item>
       <item row="5" column="0">
        <property name="text">
         <string>-</string>
        </property>
        <property name="textAlignment">
         <set>AlignHCenter|AlignVCenter|AlignCenter</set>
        </property>
       </item>
       <item row="5" column="1">
        <property name="text">
         <string>-</string>
        </property>
        <property name="textAlignment">
         <set>AlignHCenter|AlignVCenter|AlignCenter</set>
        </property>
       </item>
       <item row="5" column="2">
        <property name="text">
         <string>-</string>
        </property>
        <property name="textAlignment">
         <set>AlignHCenter|AlignVCenter|AlignCenter</set>
        </property>
       </item>
       <item row="5" column="3">
        <property name="text">
         <string>-</string>
        </property>
        <property name="textAlignment">
         <set>AlignHCenter|AlignVCenter|AlignCenter</set>
        </property>
       </item>
       <item row="5" column="4">
        <property name="text">
         <string>-</string>
        </property>
        <property name="textAlignment">
         <set>AlignHCenter|AlignVCenter|AlignCenter</set>
        </property>
       </item>
       <item row="5" column="5">
        <property name="text">
         <string>-</string>
        </property>
        <property name="textAlignment">
         <set>AlignHCenter|AlignVCenter|AlignCenter</set>
        </property>
       </item>
       <item row="5" column="6">
        <property name="text">
         <string>-</string>
        </property>
        <property name="textAlignment">
         <set>AlignHCenter|AlignVCenter|AlignCenter</set>
        </property>
       </item>
       <item row="5" column="7">
        <property name="text">
         <string>-</string>
        </property>
        <property name="textAlignment">
         <set>AlignHCenter|AlignVCenter|AlignCenter</set>
        </property>
       </item>
       <item row="5" column="8">
        <property name="text">
         <string>-</string>
        </property>
        <property name="textAlignment">
         <set>AlignHCenter|AlignVCenter|AlignCenter</set>
        </property>
       </item>
       <item row="5" column="9">
        <property name="text">
         <string>-</string>
        </property>
        <property name="textAlignment">
         <set>AlignHCenter|AlignVCenter|AlignCenter</set>
        </property>
       </item>
       <item row="5" column="10">
        <property name="text">
         <string>-</string>
        </property>
        <property name="textAlignment">
         <set>AlignHCenter|AlignVCenter|AlignCenter</set>
        </property>
       </item>
       <item row="5" column="11">
        <property name="text">
         <string>-</string>
        </property>
        <property name="textAlignment">
         <set>AlignHCenter|AlignVCenter|AlignCenter</set>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MixerCurve</class>
   <extends>QWidget</extends>
   <header location="global">mixercurve.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
